tests/tests.js

```diff
@@ -106,237 +99,252 @@ define(['jquery', 'zimArchive', 'zimDirEntry', 'util', 'utf8'],
-                    start();
+                    done();
                 }
-            }).fail(errorHandlerAsyncTest);
+            });
         });
-        asyncTest("Stylesheet '-/s/style.css' can be loaded", function() {
-            expect(4);
+        QUnit.test("Stylesheet '-/s/style.css' can be loaded", function(assert) {
+            var done = assert.async();
+            
+            assert.expect(4);
             localZimArchive.getTitleByName("-/s/style.css").then(function(title) {
-                ok(title !== null, "Title found");
+                assert.ok(title !== null, "Title found");
                 if (title !== null) {
-                    equal(title.url, "-/s/style.css", "URL is correct.");
+                    assert.equal(title.url, "-/s/style.css", "URL is correct.");
                     localZimArchive.readBinaryFile(title, function(title, data) {
-                        equal(data.length, 104495, "Data length is correct.");
+                        assert.equal(data.length, 104495, "Data length is correct.");
                         data = utf8.parse(data);
                         var beginning = "\n/* start http://en.wikipedia.org/w/load.php?debug=false&lang=en&modules=site&only=styles&skin=vector";
-                        equal(data.slice(0, beginning.length), beginning, "Content starts correctly.");
-                        start();
+                        assert.equal(data.slice(0, beginning.length), beginning, "Content starts correctly.");
+                        done();
                     });
                 } else {
-                    start();
+                    done();
                 }
-            }).fail(errorHandlerAsyncTest);
+            });
         });
-        asyncTest("Javascript '-/j/local.js' can be loaded", function() {
-            expect(4);
+        QUnit.test("Javascript '-/j/local.js' can be loaded", function(assert) {
+            var done = assert.async();
+            assert.expect(4);
             localZimArchive.getTitleByName("-/j/local.js").then(function(title) {
-                ok(title !== null, "Title found");
+                assert.ok(title !== null, "Title found");
                 if (title !== null) {
-                    equal(title.url, "-/j/local.js", "URL is correct.");
+                    assert.equal(title.url, "-/j/local.js", "URL is correct.");
                     localZimArchive.readBinaryFile(title, function(title, data) {
-                        equal(data.length, 41, "Data length is correct.");
+                        assert.equal(data.length, 41, "Data length is correct.");
                         data = utf8.parse(data);
                         var beginning = "console.log( \"mw.loader";
-                        equal(data.slice(0, beginning.length), beginning, "Content starts correctly.");
-                        start();
+                        assert.equal(data.slice(0, beginning.length), beginning, "Content dones correctly.");
```

Comment:
I suppose the starts->dones modification in the string is a mistake due to a search/replace?